From 0b3ab7d432703155145e0b9d41d7f618b7cd25e3 Mon Sep 17 00:00:00 2001
From: Harry Wentland <harry.wentland@amd.com>
Date: Tue, 2 Feb 2016 11:28:08 -0500
Subject: [PATCH 0759/1110] drm/amd/dal: Remove double call to write payload
 allocation table

This was missed in previous change to only call payload
allocation table when in mst_mode.

Signed-off-by: Harry Wentland <harry.wentland@amd.com>
Acked-by: Harry Wentland <harry.wentland@amd.com>
---
 drivers/gpu/drm/amd/dal/dc/core/dc_link.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/drivers/gpu/drm/amd/dal/dc/core/dc_link.c b/drivers/gpu/drm/amd/dal/dc/core/dc_link.c
index 2928724..2db5147 100644
--- a/drivers/gpu/drm/amd/dal/dc/core/dc_link.c
+++ b/drivers/gpu/drm/amd/dal/dc/core/dc_link.c
@@ -1562,11 +1562,6 @@ static enum dc_status deallocate_mst_payload(struct core_stream *stream)
 				&stream->public,
 				&proposed_table,
 				false);
-	dm_helpers_dp_mst_write_payload_allocation_table(
-		stream->ctx,
-		&stream->public,
-		&proposed_table,
-		false);
 
 	update_mst_stream_alloc_table(link, stream, &proposed_table);
 
-- 
2.7.4

